modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/EpoLowCoverage.pm

```diff
@@ -0,0 +1,436 @@
+               },
+    -flow_into => {
+               '2->A' => [ 'low_coverage_genome_alignment' ],
+               'A->1' => [ 'delete_alignment' ],
+              },
+    -rc_name => '4Gb_job',
+    },
+    
+    # -------------------------------------------------------------[Load species tree]--------------------------------------------------------
+    {   -logic_name    => 'make_species_tree',
+    -module        => 'Bio::EnsEMBL::Compara::RunnableDB::MakeSpeciesTree',
+    -parameters    => { 
+                               'species_tree_input_file' => $self->o('binary_species_tree'),
+              },
+    -rc_name => '1Gb_job',
+    -flow_into => 'create_default_pairwise_mlss',
+    },
+    ];
+}
+
+sub pipeline_analyses_epo2x_alignment {
+	my ($self) = @_;
+
+    return [
+    
+    # -------------------------------------------[Set conservation score method_link_species_set_tag ]------------------------------------------
+        { -logic_name => 'set_gerp_mlss_tag',
+          -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+          -parameters => {
+                          'sql' => [
+                              'INSERT INTO method_link_species_set_tag (method_link_species_set_id, tag, value) VALUES (#cs_mlss_id#, "msa_mlss_id", ' . $self->o('low_epo_mlss_id') . ')',
+                              'INSERT INTO method_link_species_set_tag (method_link_species_set_id, tag, value) VALUES (#ce_mlss_id#, "msa_mlss_id", ' . $self->o('low_epo_mlss_id') . ')',
+                          ],
+                         },
+          -flow_into => [ 'set_mlss_tag' ],
+        },
+
+        { -logic_name => 'set_mlss_tag',
+          -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+          -parameters => {
+                          'sql' => [
+                              'INSERT INTO method_link_species_set_tag (method_link_species_set_id, tag, value) VALUES (#low_epo_mlss_id#, "base_mlss_id", #base_epo_mlss_id#)',
+                          ],
+                         },
+          -flow_into => {
+                         1 => [ 'set_internal_ids' ],
+                        },
+        },
+    
+    # -----------------------------------[Create a list of pairwise mlss found in the default compara database]-------------------------------
+    {   -logic_name => 'create_default_pairwise_mlss',
+    -module     => 'Bio::EnsEMBL::Compara::RunnableDB::EpoLowCoverage::FindPairwiseMlssLocation',
+    -parameters => {
+            'new_method_link_species_set_id' => $self->o('low_epo_mlss_id'),
+            'base_method_link_species_set_id' => $self->o('base_epo_mlss_id'),
+            'pairwise_location' => $self->o('pairwise_location'),
+            'base_location' => $self->o('epo_db'),
+               },
+    -flow_into => {
+               1 => WHEN( '#run_gerp#' => [ 'set_gerp_neutral_rate' ],
+                                   ELSE [ 'import_alignment' ] ),
```

Comment:
Aesthetic reasons - the pipeline graph looks less crazy when all analyses writing to `pipeline_wide_parameters` are close to each other. Otherwise, you've got arrows sprawling all around the graph.